ewkcoffea/modules/objects_wwz.py

```diff
@@ -17,32 +17,40 @@ def get_cleaned_collection(obj_collection_a,obj_collection_b,drcut=0.4):
 ######### WWZ 4l analysis object selection #########
 
 # WWZ preselection for electrons
-def is_presel_wwz_ele(ele,tight):
+def is_presel_wwz_ele(ele,year,tight):
     mask = (
         (ele.pt               >  get_ec_param("wwz_pres_e_pt")) &
         (abs(ele.eta)         <  get_ec_param("wwz_pres_e_eta")) &
         (abs(ele.dxy)         <  get_ec_param("wwz_pres_e_dxy")) &
-        (abs(ele.dz)          <  get_ec_param("wwz_pres_e_dz")) &
+        (abs(ele.dz)          <  get_ec_param("wwz_pres_e_dz"))
+    )
+    mask_run2 = (
+        (ele.miniPFRelIso_all < get_ec_param("wwz_pres_e_miniPFRelIso_all")) &
         (abs(ele.sip3d)       <  get_ec_param("wwz_pres_e_sip3d")) &
-        (ele.miniPFRelIso_all <  get_ec_param("wwz_pres_e_miniPFRelIso_all")) &
         (ele.lostHits         <= get_ec_param("wwz_pres_e_lostHits"))
     )
-    if tight: mask = (mask & ele.convVeto & (ele.tightCharge == get_ec_param("wwz_pres_e_tightCharge")))
-    return mask
+    if ("2022" in year): mask_return = (mask & ele.mvaIso_WP80)
```

Comment:
I think this would be more clear with an if an and else, along with the statements being on their own lines. I.e.:
```
if (2022 in year):
    stuff
else:
    other stuff